src/environment.js

```diff
@@ -270,8 +319,13 @@ class Environment {
     switch (this.ci) {
       case 'travis':
         return this._env.TRAVIS_BUILD_NUMBER;
-      case 'jenkins':
+      case 'jenkins-prb':
         return this._env.BUILD_NUMBER;
+      case 'jenkins':
+        if (this._env.BUILD_TAG) {
+          return utils.reverseString(this._env.BUILD_TAG).substring(0, 60);
```

Comment:
This makes me think we should do this for TravisCI too since clicking `rebuild` on parallel builds goes 💥 for them too.